modules/tinymce/src/models/dom/main/ts/table/actions/Clipboard.ts

```diff
@@ -55,7 +56,7 @@ const registerEvents = (editor: Editor, actions: TableActions): void => {
           });
 
           const isTable = SugarNode.isTag('table');
-          if (elements.length === 1 && isTable(elements[0])) {
+          if (shouldMergeContentOnPaste(editor) && elements.length === 1 && isTable(elements[0])) {
```

Comment:
I think we settled on namespaced imports for all options. It reads a bit better `Options.shouldMergeContentOnPaste` you instantly see that it's an option not a predicate that say checks something.